run_dir/design/project_samples.html

```diff
@@ -164,6 +164,7 @@
             </thead>
             <tbody>
               <tr><td>Sample type</td><td id="sample_type"></td></tr>
+              <tr><td>Number of samples (ordered)</td><td id="sample_units_ordered"></td></tr>
```

Comment:
No, I mean - why do we need the brackets? Can it not just be `Number of samples ordered`? Seems simpler / clearer.